pvapy/cli/adSimServer.py

```diff
@@ -531,6 +638,8 @@ def main():
     parser.add_argument('-mm', '--mmap-mode', action='store_true', dest='mmap_mode', default=False, help='Use NumPy memory map to load the specified input file. This flag typically results in faster startup and lower memory usage for large files.')
     parser.add_argument('-hds', '--hdf-dataset', dest='hdf_dataset', default=None, help='HDF5 dataset path. This option must be specified if HDF5 files are used as input, but otherwise it is ignored.')
     parser.add_argument('-hcm', '--hdf-compression-mode', dest='hdf_compression_mode', default=False, action='store_true', help='Use compressed data from HDF5 file. By default, data will be uncompressed before streaming it.')
+    parser.add_argument('-aff', '--alternate-file-format', type=int, dest='alternate_file_format', default=0, help='alternate file format (not numpy or HDF5). Must be set to >0 if files used are not NumPy or HDF5. (default: 0)')
```

Comment:
It looks like alternate file format can be either true/or false. If this is the case, we could simply have alternate_file_format set to False, and then use 'store_true' action on the command line option rather than forcing user to supply integer.